src/lib.rs

```diff
@@ -203,23 +203,58 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
+    MissingFont(FontDesc),
+
+    /// Tried to get size metrics from a Face that didn't have a size.
+    MissingSizeMetrics,
```

Comment:
What does "missing" even mean here? Is that actually the error, that the metrics are missing for the font/glyph? I don't think that's the case.

Also not sure what anyone is supposed to do with this error, over just a generic `PlatformError`? Seems like there's no way to handle it anyways and it should basically never happen?

Reply:
I think I've ported it from our FreeType backend, I think it's what they are doing though. Well, there's a way to handle it, to request other font.

Reply:
It seems like all the underlying API just returns `Option` for missing metrics, and I feel like you can load a font, but which has something strange with metrics going on, so I'd keep that error for now...

Reply:
Not sure what you're talking about. You mean the other APIs (linux/macOS) return `Option`?

Reply:
Yeah, there's no error to map, you can generate something though.